sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Customizations/Models/TelemetryItem.cs

```diff
@@ -18,24 +18,24 @@ internal partial class TelemetryItem
         {
             if (activity.ParentSpanId != default)
             {
-                Tags[ContextTagKeys.AiOperationParentId.ToString()] = activity.ParentSpanId.ToHexString();
+                Tags[ContextTagKeys.AiOperationParentId.ToString()] = activity.ParentSpanId.ToHexString().Truncate(SchemaConstants.Tags_AiOperationParentId_MaxLength);
```

Comment:
Do we even need a truncation here. Number of characters is defined in W3C spec. Same for traceid

Reply:
I could go either way.
We thought the SDK Version would never need truncation, until we discovered the bad data.

I think we need the truncation here to guard against losing telemetry. 
But a counter argument might be that no data is better than bad data.

Reply:
Comparing this with sdkversion wouldn't be apt. This is well defined in spec, whereas sdkversion does just relies on some random name.

Reply:
okay, i'll get this removed for the trace/span ids